### ISSUE TITLE
Minimizing a note doesn't trigger onChange event

### ISSUE THREAD

**Issue by blackiceua** [txusko/PostItAll#45]
I'll try to positioning notes in a column after minimization. As I see, minimizing a note doesn't trigger onChange event.
Does it works as expected?

**Comment by txusko** [comment]
Yeah, you right. It should trigger the `onChange` event. I'll try to add a fix asap.
